packages/ui-color-picker/src/ColorMixer/__tests__/ColorMixer.test.tsx

```diff
@@ -128,4 +146,1591 @@ describe('<ColorMixer />', () => {
+      const onChange = stub()
+
+      await mount(
+        <ColorMixer
+          withAlpha
+          value="#80404100"
+          {...testInputLabels}
+          {...testScreenReaderLabels}
+          onChange={onChange}
+        />
+      )
+
+      const component = await ColorMixerLocator.find()
+      const alphaSlider = await component.findAlphaSlider()
+      const indicator = await alphaSlider.find('[class*=__indicator]')
+      const pos_1 = indicator.getBoundingClientRect()
+      await alphaSlider.keyDown('ArrowLeft')
+      const pos_2 = indicator.getBoundingClientRect()
+      expect(pos_2.x).to.be.eq(pos_1.x)
+      expect(onChange).to.have.been.calledOnce()
+    })
+
+    it('the alpha slider does not show when withAlpha is false', async () => {
+      const onChange = stub()
+
+      await mount(
+        <ColorMixer
+          withAlpha={false}
+          value="#80404100"
+          {...testInputLabels}
+          {...testScreenReaderLabels}
+          onChange={onChange}
+        />
+      )
+
+      const component = await ColorMixerLocator.find()
+      expect(
+        await component.findAlphaSlider({ expectEmpty: true })
+      ).to.not.exist()
+    })
+
+    it('the alpha slider does not show when withAlpha is false', async () => {
+      const onChange = stub()
+
+      await mount(
+        <ColorMixer
+          value="#80404100"
+          {...testInputLabels}
+          {...testScreenReaderLabels}
+          onChange={onChange}
+        />
+      )
+
+      const component = await ColorMixerLocator.find()
+      expect(
+        await component.findAlphaSlider({ expectEmpty: true })
+      ).to.not.exist()
+    })
+
+    it('should set the disabled attribute when `disabled` is set', async () => {
+      const onChange = stub()
+
+      await mount(
+        <ColorMixer
+          withAlpha
+          disabled
+          value="#80404100"
+          {...testInputLabels}
+          {...testScreenReaderLabels}
+          onChange={onChange}
+        />
+      )
+
+      const component = await ColorMixerLocator.find()
+      const alpharSlider = await component.findAlphaSlider()
+      expect(await alpharSlider.find('[disabled]')).to.exist()
+    })
+  })
+
+  describe('color palette', () => {
+    describe('should change the color when receive mousedown', () => {
+      it('inside the palette', async () => {
+        const onChange = stub()
+        await mount(
+          <ColorMixer
+            {...testInputLabels}
+            {...testScreenReaderLabels}
+            onChange={onChange}
+          />
+        )
+        const component = await ColorMixerLocator.find()
+        const colorPalette = await component.findColorPalette()
+        const rect = await colorPalette.getBoundingClientRect()
+        await colorPalette.mouseDown({
+          clientX: rect.x + rect.width / 2,
+          clientY: rect.y + rect.height / 2
+        })
+        expect(onChange).to.have.been.called()
```

Comment:
I'd also test for the parameters of the `onChange` function. For examples look for tests using `calledWithMatch` or `calledWith`